api/src/main/resources/liquibase.xml

```diff
@@ -712,4 +712,22 @@
             ALTER TABLE patient_appointment MODIFY end_date_time datetime default null;
         </sql>
     </changeSet>
+    <changeSet id="add-patient-appointment-encounter-map-table-20240429" author="mgoodrich">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="patient_appointment_encounter_map" />
+            </not>
+        </preConditions>
+        <comment>Add the map table to link appointments to encounter</comment>
+        <sql>
```

Comment:
We should not use raw sql for this, especially not when it contains backticks, which seems very mysql-specific.  Liquibase has a <createTable...> element, with correponding <column> elements, as well as a <addForeignKeyContstraint> element.  These should be used preferably.  There is also no need for the `IF NOT EXISTS` statement given the precondition that is on the overall changeset.

Reply:
Fair enough